js-exercises/sequential-promise/sequentialPromise.js

```diff
@@ -0,0 +1,13 @@
+const sequentialPromise = async promises => {
+  let result = "";
+  for (let promise of promises) {
+    await promise(result).then(value => {
+      result = value;
+      return result;
+    });
+  }
```

Comment:
you can't assume only promises array will come. Handling checks whether expected input came or not will help your program